test/UnitTest/Microsoft.AspNet.OData.Test.Shared/PublicApi/PublicApiTest.cs

```diff
@@ -40,7 +40,7 @@ public void TestPublicApi()
                 String.Format("Base line file {1} and output file {2} do not match, please check.{0}" +
                 "To update the baseline, please run:{0}{0}" +
                 "copy /y \"{2}\" \"{1}\"", Environment.NewLine,
-                @"test\UnitTest\Microsoft.AspNet.OData.Test\PublicApi\" + BaseLineFileName,
+                @"test\UnitTest\Microsoft.AspNetCore.OData.Test\PublicApi\" + BaseLineFileName,
```

Comment:
Sadly, this is a little more complicated given that the baselines are in different folders. I also have a fix for this which creates a BaseLineFileFolder value that each test dll can set and allow the instruction to point to different locations. You can feel free to copy mine or leave this out of the change. See #1433

Reply:
Ok, reverted it then, better having your fix!